app/api/v1/users.rb

```diff
@@ -0,0 +1,17 @@
+class V1::Users < Grape::API
+  before do
+    authenticate!
+  end
+
+  desc 'Return all users', { tags: ['user'] }
+  params do
+    optional :created_at, type: DateTime
+  end
+  paginate
+  get '/users/' do
+    users = User.all
+    users = users.where("created_at > ?", params[:created_at]) if params[:created_at]
```

Comment:
Couldn't you use a conditional block here? You're always fetch all users then "filtering" with the created params.
Something like:
```ruby
user ||=
 if params[:created_at]
  User.where("created_at > ?", params[:created_at])
 else
  User.all
 end
```